release-notes/7.0/preview/api-diff/preview5/Microsoft.NETCore.App/7.0-preview5_System.ComponentModel.md

```diff
@@ -0,0 +1,11 @@
+# System.ComponentModel
+
+``` diff
+ namespace System.ComponentModel {
+     public sealed class TypeDescriptor {
+         public static Type ComObjectType { get; }
+         public static Type InterfaceType { get; }
```

Comment:
These APIs just got "[EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Advanced)]" attributes put on them. They are not new.

https://github.com/dotnet/runtime/pull/68432/files#diff-6466df6aea2f15578eae754246e298f9fd3739cc4e81eb7736977ba7efde18e0R1387

Is there a bug in the diff tool here that it should be showing the new attributes, and not indicating these APIs are completely new?

cc @ericstj who made the PR

Reply:
Yep, I mentioned it in the description. 🙂

Reply:
I'll have to manually edit these.